extensions/avro/src/main/java/com/hazelcast/jet/avro/AvroSourceBuilder.java

```diff
@@ -82,6 +82,18 @@
         return this;
     }
 
+    /**
+     * Sets the preferred local parallelism of the source. Default values is
+     * {@code 4}.
+     *
+     * @since 4.2
+     */
+    @Nonnull
+    public AvroSourceBuilder<D> preferredLocalParallelism(int preferredLocalParallelism) {
```

Comment:
do we need this option? I think just to change the default would have been enough. Do we have this in any other source?

Reply:
I don't think we have, I'll remove then. So wdyt about `4` as default ?
Should we set it only for Avro source ? or both (avro and file) ?